lib/ServerSai.cpp

```diff
@@ -1237,7 +1237,7 @@ sai_status_t ServerSai::processBulkCreateEntry(
         attr_lists[idx] = attributes[idx]->get_attr_list();
     }
 
-    switch (objectType)
+    switch ((int)objectType)
```

Comment:
Why is this cast to `int` needed?

Reply:
probably because OT can contain extended object type which is different enum and compiler can error on that

Reply:
Thanks, Kamil. This is correct.